pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
 
 import (
+	"fmt"
 	"testing"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	. "github.com/kubernetes-incubator/service-catalog/pkg/controller/apiclient"
+	"github.com/kubernetes-incubator/service-catalog/pkg/controller/apiclient/mem"
+	kapi "k8s.io/kubernetes/pkg/api"
 )
 
-// TODO: implement these (https://github.com/kubernetes-incubator/service-catalog/issues/134)
-func TestGetServicePlanInfo(t *testing.T) {
+func TestGetServicePlanInfoEmpty(t *testing.T) {
+	apiclient := mem.NewAPIClient()
+	if _, _, _, err := GetServicePlanInfo(apiclient.ServiceClasses(), "service", ""); err == nil {
+		t.Error("Expected err on GetServicePlanInfo with no plan with empty apiclient")
+	}
+	if _, _, _, err := GetServicePlanInfo(apiclient.ServiceClasses(), "service", "plan"); err == nil {
+		t.Error("Expected err on GetServicePlanInfo with plan with empty apiclient")
+	}
+}
+
+// test service classes with single plan
+func TestGetServicePlanInfoOne(t *testing.T) {
+	apiclient := mem.NewAPIClient()
+	svcClss := createServiceClassesWithServicePlans(2, 1)
+	apiclient.ServiceClasses().Create(&svcClss[0])
+
+	// test that getting with empty plan name works
+	if err := testGetServicePlanInfo(apiclient, &svcClss[0], &svcClss[0].Plans[0], ""); err != nil {
+		t.Error(err)
+	}
+
+	// test that getting with correct plan name works
+	if err := testGetServicePlanInfo(apiclient, &svcClss[0], &svcClss[0].Plans[0],
+		svcClss[0].Plans[0].Name); err != nil {
```

Comment:
Just a general comment about line breaking throughout this. Go doesn't have a column restriction so this and others can be one line